frontends/web/src/routes/account/receive/receive.tsx

```diff
@@ -48,6 +48,7 @@ interface State {
     paired: boolean | null;
     // index into `availableScriptTypes`, or 0 if none are available.
     addressType: number;
+    addressDialog: { addressType: number } | false;
```

Comment:
why `false` and not `undefined`?

Reply:
I had boolean first and that became the obj+false, `undefined` makes more sense, I'll update the PR.